eth/backend.go

```diff
@@ -161,13 +165,17 @@ func (b *backend) PendingCallContract(ctx context.Context, msg ethereum.CallMsg)
 }
 
 func (b *backend) retryRemoteCall(remoteCall func() ([]byte, error)) (out []byte, err error) {
-	count := 3    // consider making this a package-level global constant
 	retry := true // consider making this a package-level global constant
 
-	for i := 0; i < count && retry; i++ {
+	for i := 0; i < maxRemoteCallRetries && retry; i++ {
 		out, err = remoteCall()
-		if err != nil && (err.Error() == "EOF" || err.Error() == "tls: use of closed connection") {
-			glog.V(4).Infof("Retrying call to remote ethereum node")
+		if err != nil && isRetryableRemoteCallError(err) {
+			glog.V(4).Infof("Retrying call to remote ethereum node err=%v", err)
+			// We could use the backoff package to configure an exponential backoff here, but it makes it
+			// more difficult to propagate the result from the call if the remote call is successful because
+			// the backoff functions can only return a single error type.
+			// So, to keep things simple just sleep here for 1 second before trying again.
+			time.Sleep(remoteCallRetrySleep)
```

Comment:
I think you could implement exponential backoff just by multiplying it with `i`.

```
time.Sleep(i * remoteCallRetrySleep)
```

Then, you could make `remoteCallRetrySleep` slightly smaller, like `500ms`.

Reply:
Good point. I ended up implementing a linear backoff by multiplying `remoteCallRetrySleep` by `i` in [c04c2ee](https://github.com/livepeer/go-livepeer/pull/2563/commits/c04c2ee7eff5f86b74adad238fac101e474f926b) based on your suggestion which feels good enough.